tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -252,6 +252,9 @@ TEST_F(EnergyPlusFixture, DXCoils_Test1)
     DXCoilPartLoadRatio.allocate(1);
     DXCoilFanOpMode.allocate(1);
 
+    DataLoopNode::Node.allocate(1);
+    DXCoil(CoilIndex).AirOutNode = 1;
+
```

Comment:
Wow, previously no Node calls inside the Calc routine, I wouldn't have guessed that but I guess Init sets up node data variables and uses them in Calc. Wonder what the node data variable shows in the Calc routine. Oh well, unit test passes what it's intended to do.

Reply:
Other than the condenser inlet node, CalcDoe2DXCoil doesn't read or write to any nodes. It uses variables like `DXCoil(DXCoilNum).InletAirTemp`.